docs/request.md

```diff
@@ -27,13 +27,19 @@ Makes XHR (aka AJAX) requests, and returns a [promise](promise.md)
 m.request({
 	method: "PUT",
 	url: "/api/v1/users/:id",
-	params: {id: 1, name: "test"}
+	params: {id: 1},
+	body: {name: "test"}
 })
 .then(function(result) {
 	console.log(result)
 })
 ```
 
+The above code will make a `PUT` request to `/api/v1/users/1` with a JSON body of `{"name": "test"}`.
```

Comment:
This part will be explained [later](https://github.com/MithrilJS/mithril.js/blob/c31a72a843f3f67128517af53e998d0038ca7531/docs/request.md#dynamic-urls), and the signature provides enough context to be able to interpret most of this on its own. This section is just a "Cliff's Notes" version to show what it looks like, so the extra clutter is just distracting.

Reply:
Updated.